src/terminal.rs

```diff
@@ -105,7 +105,7 @@ impl Terminal {
                     command.args(["-a", "Topgrade", "Topgrade"]);
                     command.arg(message.as_ref());
                     if let Err(err) = command.output_checked() {
-                        log::error!("{err:?}");
+                        tracing::error!("{err:?}");
```

Comment:
Couln't this one be `error!(` only since line 17 already use `tracing::error`?

This comment also apply to other times `log::` was called directly, it would be more consistent to have them all changed to unqualified call?

Reply:
A lot of these changes were find/replace'd, so they may not be super consistent.